src/main/java/com/minecolonies/achievements/ModAchievements.java

```diff
@@ -136,6 +137,45 @@
     public static final Achievement achievementUpgradeFarmerMax = new MineColoniesAchievement("upgrade.farmer.max", 2, 6, ModBlocks.blockHutFarmer,
                                                                                                achievementBuildingFarmer).registerStat();
 
+    /**
+     * Upgrade a guard to lv 1.
+     */
+    public static final Achievement achievementBuildingGuard   = new MineColoniesAchievement("upgrade.guard.first", 0, 7, ModBlocks.blockHutGuardTower,
+                                                                                               achievementTownhall).registerStat();
+    /**
+     * Max out a guard.
+     */
+    public static final Achievement achievementUpgradeGuardMax = new MineColoniesAchievement("upgrade.guard.max", 2, 7, ModBlocks.blockHutGuardTower,
+                                                                                               achievementBuildingGuard).registerStat();
+
+    /**
+     * Kill one mob
+     */
+    public static final Achievement achievementKillOneMob = new MineColoniesAchievement("guard.mobkill.one", 2, 8, Items.BONE, achievementBuildingGuard).registerStat();
+
+    /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)